packages/algorand-js/src/types.ts

```diff
@@ -70,3 +73,37 @@ export enum EncodingType {
   HEX,
   UTF8
 }
+
+export interface AccountsMap {
+  [addr: string]: StoreAccount
+}
+
+export interface State {
+  accounts: Map<string, StoreAccount>
+  globalApps: Map<number, SSCParams>
+}
+
+// describes interpreter's local context (state + txns)
+export interface Context {
+  state: State
+  tx: Txn // current txn
+  gtxs: Txn[] // all transactions
+  args: Uint8Array[]
+}
+
+// represent account used in tests and by the context
+// NOTE: custom notations are used rather than SDK AccountState notations
+export interface StoreAccount {
+  address: string
+  assets: AccountAssetInfo[]
+  amount: number
+  appsLocalState: AppLocalState[]
+  appsTotalSchema: SSCSchemaConfig
+  createdApps: CreatedApps[]
+  createdAssets: CreatedAssets[]
+  account?: Account
+
+  balance: () => number
+  getLocalState: (appId: number, key: Uint8Array) => StackElem | undefined
+  updateLocalState: (appId: number, key: Uint8Array, value: StackElem) => AppLocalState[]
+}
```

Comment:
`algorand-js` shouldn't have a dependency to algob.
The dependency graph is that algob import algorand-js, not other way around.
So, for functional tests, where we create an algob project, we should use the algob package tests  (with a fixture project).
For simple runtime testing (this is what you are doing in the tests you created) we don't need a fixture project. You just set some accounts and that's it.

Reply:
removed dependency